### DISCUSSION TITLE
Blazorwebview - Android emulator - bypass SSL verification

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#25798] [Q&A]
Hello.

I know how to bypass the SSL-verification of the android emulator when using an http client (and requesting data from localhost).
I now want to display some images using a regular img tag in blazor from my local dev-host.
Does anyone know how get around the SSL verification?

Thanks a lot for your help!

**Comment by timdinh-grex** [comment]
you could use vscode port forwarding feature or ngrok (a reverse proxy)
